docs/reference/modules/module.md

```diff
@@ -0,0 +1,103 @@
+
+:::warning
+**This module should almost never be configured directly in project code.** In other words, there should almost never be a `modules/@apostrophecms/module/index.js` file in a project codebase or `@apostrophecms/module: {}` line in an `app.js` file. The only reason to do so would be to add a feature that every other module will need.
+
+**Instead, configure the options below on other modules as needed** (e.g., configure the `alias` option on a module to create an easy reference to *that* module). Similarly, run the methods below as needed from the most appropriate module (e.g., use `self.email` from a contact page-type module).
+:::
+
+## Options
+
+|  Property | Type | Description |
+|---|---|---|
+|`alias` | String | A name to use for quick access to the module on the `apos` object (e.g., `'book'` to access a module on `self.apos.book`). Otherwise the longer syntax is necessary (e.g., `self.apos.modules.book`). |
+|`templateData` | Object | An object of properties to include on the `data` object in templates belonging to that module. |
+
+## Featured methods
+
+The following methods belong to this module and may be useful in project-level code. As noted above, *they should be used via another, more appropriate module,* rather than directly called from this module. See the [source code](https://github.com/apostrophecms/apostrophe/blob/main/modules/%40apostrophecms/module/index.js) for all methods that belong to this module.
+<!-- Some are used within the module and would just create noise here. -->
+
+### `async render(req, template, data)`
+
+Returns rendered HTML for a template with the data provided. You must pass `req`, a request object, as the first argument. The `template` argument should be the name of a template file in the module's `views` directory.
+
+All properties of `data` can be used in Nunjucks templates as properties of the `data` object. This argument may be omitted to include no additional data.
+
+### `async renderString(req, string, data)`
+
+Returns rendered HTML for a Nunjucks-style string with the data provided. You must pass `req`, a request object, as the first argument. The `string` argument will be used as the template itself for rendering (not the template filename).
+
+All properties of `data` can be used in Nunjucks templates as properties of the `data` object. This argument may be omitted to include no additional data.
+
+### `async send(req, template, data)`
+
+The `send()` method renders a template with data as with the [`render()`](#async-render-req-template-data), then sends the rendered HTML as a response to the request (`req`).
+
+### `async sendPage(req, template, data)`
+
+Similarly to the [`send()`](#async-send-req-template-data) method renders a template and sends the rendered HTML as a response to the request (`req`). Where `send()` is used to render general template files, `sendPage()` is specifically used to render and send *full pages* for Apostrophe projects.
```

Comment:
Grammar.

Reply:
There are some pretty critical details in the inline documentation of this method. In particular the developer is meant to use:

`{% extends data.outerLayout %}`

In their template, otherwise calling this method doesn't accomplish much over `send`.

The inline documentation in the source is wrong on this point, it leaves out the `data.` and says `extend` rather than `extends`.